src/main/org/deidentifier/arx/metric/v2/MetricMDNMLoss.java

```diff
@@ -305,6 +381,16 @@ protected void initializeInternal(final DataManager manager,
         
         // Save domain shares
         this.shares = manager.getDomainShares();
+
+        if (config.isPrivacyModelSpecified(EDDifferentialPrivacy.class)) {
```

Comment:
Könnte man diesen sich ständig wiederholenden Codeblock nicht in der entsprechenden Methode der Superklasse für alle Instanzen von InformationLoss lösen?

Reply:
Ich habe den Codeblock hochgezogen in die Methode initializeInternal der Superklasse AbstractMetricMultiDimensional. Neben den Subklassen von AbstractMetricMultiDimensional haben wir noch zwei von AbstractMetricSingleDimensional abgeleitete Klassen, von denen allerdings nur eine diesen Codeblock hat. Daher hab ich ihn in diesem Fall in der abgeleiteten Klasse gelassen.